core/chains/evm/config/toml/defaults/Hedera_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '295'
+ChainType = 'hedera'
+#considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+#we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
+MinIncomingConfirmations = 1
+#turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
+NoNewHeadsThreshold = '0'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+Mode = 'SuggestedPrice'
+# since Hedera dont have mempool and there's no way for a node to front run or a user to bribe a node to submit the transaction earlier than it's consensus timestamp
+# but they have automated congesting pricing throttling which would mean at high sustained level the gasPrice itself could be increased to prevent malicious behaviour.
+# # disabling the Bumpthreshold as TXM now implicity handles the bumping after checking on-chain nonce & re-broadcast for Hedera chain type
```

Comment:
@amit-momin Could you ensure this gas setting works for Hedera, specially wrt changes you made?

Reply:
I think these look good. We wouldn't need bumping during confirmation since there's no mempool and transactions are included immediately. But we still need `BumpMin` and `BumpPercent` since our custom broadcasting logic retries 3 times with bumped prices if the nonce doesn't increment.